package.js

```diff
@@ -2,26 +2,26 @@
 Package.describe({
   name: 'dburles:mongo-collection-instances',
   summary: 'Access your Mongo instances',
-  version: '0.4.0',
+  version: '1.0.0-beta300.1',
   git: 'https://github.com/dburles/mongo-collection-instances.git'
 })
 
 Package.onUse(function (api) {
-  api.versionsFrom(['1.0', '2.3', '2.8.0'])
+  api.versionsFrom(['2.8.0', '3.0-beta.0'])
   api.use([
-    'mongo',
-    'ecmascript',
-    'lai:collection-extensions@0.4.0'])
+    'ecmascript@0.16.8 || 0.16.8-beta300.0',
+    'mongo@1.16.8 || 2.0.0-beta300.0',
+    'lai:collection-extensions@1.0.0-beta300.1'])
```

Comment:
If we're going to use `versionsFrom` then it's better not to fix the sub packages versions.

Reply:
At least `lai:collection-extensions`would have to be fixed, but for the core packages you're right, for them we would not need a specific version

Reply:
I'm fine with that compromise :+1: